sklearn/linear_model/logistic.py

```diff
@@ -802,6 +823,11 @@ def _log_reg_scoring_path(X, y, train, test, pos_class=None, Cs=10,
     y_train = y[train]
     y_test = y[test]
 
+    if sample_weight is None:
+        sample_weight_train = None
+    else:
+        sample_weight_train = sample_weight[train]
+
```

Comment:
minor:
Is it better to simply write?

    if sample_weight is not None:
        sample_weight = sample_weight[train]

and pass sample_weight